src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionsCallout.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../../../../../Models/Services/StoryUtilities';
+import ConditionsCallout from './ConditionsCallout';
+import { IConditionsCalloutProps } from './ConditionsCallout.types';
+
+const wrapperStyle = { width: '100%', height: '600px', padding: 8 };
+
+export default {
```

Comment:
this story is busted
![image](https://user-images.githubusercontent.com/57726991/197074856-c6c694d5-07ad-4866-bc89-e5b0a283b6ed.png)